README.md

```diff
@@ -35,13 +44,26 @@ To stop the server:
   - Option 1: `sudo nginx -c /usr/local/etc/nginx/nginx.conf -s stop`
   - Option 2: `sudo nginx -s stop`
 
-## Docker
+## Docker instructions
 
-To run with docker:
-1. Build the production version of this project: `ng build --prod`
-2. Build image: `docker build -t angular-nginx .`
-3. Run container: `docker run --name angular-nginx-container -d -p 8090:80 angular-nginx`
-4. Go to `localhost:8090`
+To run without using the docker-compose command:
+
+First build the image:
+
+```bash
+docker build --build-arg cert_dir=./certs -t lantern_front_end .
```

Comment:
Can you add the command used to publish the image to dockerhub?

Reply:
👍